lib/cndb/cndb.c

```diff
@@ -581,6 +581,59 @@ cndb_record_kvset_del(
     return err;
 }
 
+merr_t
+cndb_record_kvset_move(
```

Comment:
This function name is similar to the kvset add and delete functions. But this one operates on multiple kvsets. Maybe rename this to kvsets_mode, kvsetv_move or some such....

Reply:
Done, changed it to kvsetv_move..